### PR TITLE
Fs 1190 status fix

### PR DESCRIPTION
**Pull request by donshellywork**
https://digital.dclg.gov.uk/jira/browse/FS-1190

### CODE REVIEW

**Comment by gidsg** [comment]
Can you add a screenshot where we are making a visual change like this?

**Comment by donshellywork** [comment]
@gidsg I couldn't figure out how to nav to the page, i was advised to push anyway

File: app/templates/tasklist.html [hunk 1]
```diff
@@ -9,13 +9,28 @@ <h1 class="govuk-heading-xl">
...
+      <strong class="govuk-tag govuk-tag--blue">
+        In Progress
+      </strong>
       {% endif %}
-    </h2>
+      {% if completed_count == section_count %}
+      <strong class="govuk-tag govuk-tag--green">
+        Complete
+      </strong>
+      {% endif %}
```
Comment (gidsg) [inline app/templates/tasklist.html:31]: Could we use `elif` here as these should be exclusive? e.g. https://jinja.palletsprojects.com/en/3.0.x/templates/#if

**Comment by donshellywork** [comment]
Screenshot 2022-07-26 113109
Screenshot 2022-07-26 112401

File: app/templates/tasklist.html (hunk 1 above)
Comment (donshellywork) [reply app/templates/tasklist.html:31]: good point - implementing

**Review by gidsg — APPROVED** [review]
LGTM 👍

**Pull request merged**